workspaces/npm/plugins/npm/dev/index.tsx

```diff
@@ -15,13 +15,48 @@
  */
 import React from 'react';
 import { createDevApp } from '@backstage/dev-utils';
-import { npmPlugin } from '../src/plugin';
+import { npmPlugin, NpmReleases } from '../src/plugin';
+import { EntityProvider } from '@backstage/plugin-catalog-react';
+import { NpmInfoCard } from '../src/components/NpmInfoCard';
```

Comment:
If should load this then also from plugin.ts, or?

Reply:
Updated the usage and importing it from plugin.ts